04/app.js

```diff
@@ -1,5 +1,7 @@
 import React from 'react';
+import { v4 as uuid } from "uuid";
 import { createRoot } from 'react-dom/client';
+import ReactFileReader from 'react-file-reader';
```

Comment:
Jeśli korzystasz z dodatkowych paczek to pamiętaj, aby zapisać je w package.json (używać flagi `--save-dev` lub `--save`), bo inaczej osoba korzystająca z kodu ma błąd (tak jak ja ;P)

UPDATE: Pewnie przypadkowo to Ci się dodało bo nie widzę, abyś wykorzystywał go :P

![image](https://user-images.githubusercontent.com/55095257/227121336-d5044f8d-2c66-4034-a4ef-6d13c1275c2a.png)